openedx/features/ucsd_features/templates/ucsd_features/base_email.html

```diff
@@ -0,0 +1,6 @@
+<html>
```

Comment:
How is this file different from the above file of the path: openedx/features/ucsd_features/templates/openedx_caliper_tracking/base_email.html?